docs/_posts/2016-01-11-admin-manual.md

```diff
@@ -362,15 +365,16 @@ If you intend to scan Illumina runs, you will have to build `runscanner-illumina
 
 Start Tomcat on this machine.
 
-Edit `miso.properties` and set `miso.runscanner.urls` to the URL of the Run Scanner instance and restart MISO.
+Edit `$CATALINA_HOME/conf/Catalina/localhost/miso.properties` and set `miso.runscanner.urls` to the URL of the Run Scanner instance and restart MISO.
 
 It is possible to set up multiple run scanners managing different sequencers and add all the URLs to `miso.properties`.
 
-You can view the run scanner's state from the main page.
+You can view the run scanner's state from the main page of the runscanner server (http://runscanner.url:8080).
```

Comment:
Probably shouldn't specify the port. It's a product of Tomcat and other server/network config and MISO/runscanner don't affect or care what port is used